apis/kubedb/v1/types.go

```diff
@@ -204,6 +204,14 @@ type SystemUserSecretsSpec struct {
 }
 
 type SecretReference struct {
+	// +optional
+	// Two possible groups: "", virtual-secrets.dev
+	Group string `json:"group,omitempty"`
```

Comment:
The standard way is to name it `apiGroup`.  

For example, see the field name in `kubectl explain clusterrolebinding.spec.roleRef`